core/blockly.ts

```diff
@@ -723,7 +722,6 @@ export {Toolbox};
 export {ToolboxCategory};
 export {ToolboxItem};
 export {ToolboxSeparator};
-export {TouchGesture};
```

Comment:
@maribethb for breaking change decisions.

Reply:
still a breaking change, unfortunately. haven't reviewed the rest of the PR, but if it's just a rename then we should be able to add an extra export statement of the old name with the new content, plus a comment saying to remove in version 10. I don't know if we can easily add an actual deprecation warning if we're just exporting under a new name but that's ok. Then please tag this PR as a deprecating change so we can call it out in release notes.

Reply:
Reexporting sounds + looks good, I don't think there's anyway that could break people (even instanceof checks should still succeed :P ).

But thinking about this more: is there any reason for us to export `TouchGesture`? Or should we deprecate both of them? idk why an external dev would need to instantiate or reference a gesture.

Reply:
It does include the static inProgress(), which I could see being generically useful, but I'm happy to defer to your/the rest of the team's judgement.

Reply:
Keep it exported because removing it entirely would be a breaking change, not a deprecation.

Reply:
I definitely agree that the class should be exported under both names for now (for backwards compatibility). But I think long-term it'd be better to not export it at all (since idk of a use-case for it).

wrt `inProgress` I think `workspace.isDragging()` + `WidgetDiv.isVisible()` + etc covers that.

So I think we should remove both exports in v10.